sources/api/netdog/src/net_config/v3.rs

```diff
@@ -70,21 +79,30 @@ impl Validate for NetConfigV3 {
         }
 
         for (name, device) in &self.net_devices {
+            // Bonds / vlans cannot be configured via MAC address
```

Comment:
I'd like to specifically call out they cannot be configured via MAC address due to wicked not supporting this use. There may be a future where a different network backed could support this.
```suggestion
            // Bonds / vlans cannot be configured in wicked via MAC address
```